addons/hold-delete/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Hold to delete sprites",
+  "description": "Require the delete button to be held down in order to delete a sprite.",
+  "tags": ["editor"],
+  "credits": [
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
+    },
+    {
+      "name": "Samq64",
```

Comment:
There's no need to credit me here. Everything after the first commit from my PR was suggested by @WorldLanguages.